### PR TITLE
init now builds a default barebones theme

### PR DESCRIPTION
**Pull request by aslowwriter**
This PR is in response to #672 I wasn't sure what people wanted from a default theme, so I went for the most bare bones thing possible: No CSS, an `index.html` that just displays the title and any pages and their content, or some filter text if those are not available, a 404 page that just says "not found" and page.html that just lists the title and content of a page. Since they were small enough I embedded them in the `cmd::init` file for now just like the `Config.toml`, but this will likely not work for a more fleshed out basic theme. If no theme is supplied, it will look for this one. Since any other theme or template will override this it should not have an impact on most users.

I have not updated docs yet because I wanted to get some feedback on the theme beforehand. If people want adjustments to the theme or the way it is generated, I'm open to suggestions.

## Checklist:
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/getzola/zola/pulls) for the same update/change?
* [x] Are you doing the PR on the `next` branch?
* [ ] Have you created/updated the relevant documentation page(s)?

### CODE REVIEW

**Comment by aslowwriter** [comment]
A question for @Keats with the idea of backwards compatibility. If the default theme is not found, should we just rebuild it, or just fail?

File: src/cmd/init.rs [hunk 1]
```diff
@@ -131,6 +205,21 @@ fn populate(path: &Path, compile_sass: bool, config: &str) -> Result<()> {
         create_dir(path.join("sass"))?;
     }
 
+    setup_default_theme(&path.join("themes"))?;
+    Ok(())
+}
+
+fn setup_default_theme(path: &Path) -> Result<()> {
```
Comment (Keats) [inline src/cmd/init.rs:212]: I think we don't want a theme, we just want some default templates. We do have some that get automatically used in `serve` and I would prefer expanding those. Once they are in a good shape, we can write them to disk in the init.

**Review by Keats — CHANGES_REQUESTED** [review]

File: src/cmd/init.rs (hunk 1 above)
Comment (aslowwriter) [reply src/cmd/init.rs:212]: I've tried to look for them but for the life of me I can't find where those get defined. Mind pointing me in the right direction?

Comment (Keats) [reply src/cmd/init.rs:212]: https://github.com/getzola/zola/blob/master/components/utils/src/templates.rs extremely basic as you can see, https://github.com/getzola/zola/blob/master/components/utils/src/default_tpl.html for the actual template.

**Pull request closed without merging**